app/views/whitehall_migration/document_imports.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :back_link, render_back_link(href: whitehall_migration_path(@whitehall_migration)) %>
+<% title = @state ? "Document Imports: #{@state.titleize}" : "Document Imports" %>
+<% content_for :title, title %>
+
+<% if @document_imports.any? %>
+<%= render "document_imports_list", document_imports: @document_imports %>
+<% else %>
+  <p class="govuk-body">
+    There are no document imports
+    <% if @state %>
+      with state: <%= @state.titleize %>
+    <% end %>
+  </p>
+<% end %>
+
+<%
+  pages = {}
+  pages[:previous_page] = {
+    url: whitehall_migration_document_imports_path(migration_id: @whitehall_migration, page: @document_imports.prev_page, state: params['state']),
+    title: "Previous page",
+    label: "#{@document_imports.prev_page} of #{@document_imports.total_pages}"
+  } if @document_imports.prev_page
+
+  pages[:next_page] = {
+    url: whitehall_migration_document_imports_path(migration_id: @whitehall_migration, page: @document_imports.next_page, state: params['state']),
```

Comment:
You'll want to switch params[:state] to @state here and on line 19

Reply:
Thanks, have amended!